src/main.cpp

```diff
@@ -8,8 +8,10 @@
 #include "timing.h"
 
 using solver::Solver;
-using solver::SolverProfiler;
 using solver::SolverType;
+#ifdef PROFILER_ENABLED
+using solver::SolverProfiler;
```

Comment:
I think we can do away with the SolverProfiler class. I ended up just using gprof, which was more than adequate.